pom.xml

```diff
@@ -6,30 +6,22 @@
   <artifactId>assertj-parent-pom</artifactId>
   <version>2.2.4-SNAPSHOT</version>
   <packaging>pom</packaging>
+
   <name>AssertJ - Fluent assertions for java unit testing</name>
   <description>Parent POM for all AssertJ modules</description>
   <url>http://assertj.org</url>
+  <organization>
+    <name>AssertJ</name>
+    <url>joel-costigliola.github.io/assertj/index.html</url>
```

Comment:
let's use https://assertj.github.io/doc/ instead as we are (slowly) moving to the new website